test/oai/test_gemini.py

```diff
@@ -0,0 +1,150 @@
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+try:
+    from google.api_core.exceptions import InternalServerError
+
+    from autogen.oai.gemini import GeminiClient
+
+    skip = False
+except ImportError:
+    GeminiClient = object
+    InternalServerError = object
+    skip = True
+
+
+# Fixtures for mock data
+@pytest.fixture
+@pytest.mark.skipif(skip, reason="Google GenAI dependency is not installed")
```

Comment:
This is ignored by pytest, you can remove it, see: https://docs.pytest.org/en/latest/reference/reference.html#marks

### [Marks](https://docs.pytest.org/en/latest/reference/reference.html#id72)
Marks can be used to apply metadata to test functions (but not fixtures), which can then be accessed by fixtures or plugins.

Reply:
Thanks for the tops!~